### PR TITLE
PLT-3061 Load user statuses as soon as possible

### PR DESCRIPTION
**Pull request by enahum**
Same as https://github.com/mattermost/platform/pull/3121 but against master

### CODE REVIEW

**Comment by crspeller** [comment]
Rebase please.

File: webapp/components/logged_in.jsx [hunk 1]
```diff
@@ -139,6 +110,45 @@ export default class LoggedIn extends React.Component {
...
 
+        // Pervent backspace from navigating back a page
+        $(window).on('keydown.preventBackspace', (e) => {
+            if (e.which === BACKSPACE_CHAR && !$(e.target).is('input, textarea')) {
+                e.preventDefault();
+            }
+        });
+    }
+
+    componentWillMount() {
```
Comment (jwilander) [inline webapp/components/logged_in.jsx]: Can we move all this into the constructor instead of being in `componentWillMount`?

**Comment by enahum** [comment]
moved to constructor

**Comment by hmhealey** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**